docs/communications/checklist.md

```diff
@@ -48,10 +48,13 @@ nav_order: 3
 
 Update all social media with new branding and dates each year
 
-- [ ] Twitter
```

Comment:
Technically, our socials still seem to be using it. Maybe they are using Typefully.com or Buffer? 

See https://x.com/djangocon